ferrilata-devilline/Models/Error.cs

```diff
@@ -0,0 +1,40 @@
+namespace ferrilata_devilline.Models
+{
+    public class Error
+    {
```

Comment:
i think it's time to make subfolders of Models - this sounds like kind of a helper, there will be the objects that are mapped to the database, and also the data transfer objects.

Reply:
Sounds like a good idea, let's discuss on Monday.